src/config/dynamic-theme-fixes.config

```diff
@@ -546,6 +546,9 @@ INVERT
 .ms-Grid-col.ms-sm8.ms-lg4
 .ms-Grid-col.ms-sm4.ms-hiddenMdDown.pullout-textwrap
 .ms-Grid-col.ms-sm2 img
+.pullout-link-text
+.pullout-icon
```

Comment:
Too general... breaks circle check icon.
I propose `.pullout-icon:not(.ms-Icon--SkypeCircleCheck)`